samples/Sentry.Samples.NLog/Program.cs

```diff
@@ -12,7 +12,7 @@ public static class Program
 
     // DSN used by the example: 'UsingCodeConfiguration'.
     // #### ADD YOUR DSN HERE:
-    private static readonly string DsnSample = "https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.sentry.io/5428537";
+    private static readonly string DsnSample = "https://eb18e953812b41c3aeb042e666fd3b5c@o447951.ingest.us.sentry.io/5428537";
```

Comment:
low: my other question made me notice that we've 31 samples, but only changes within 30 samples here

`Sentry.Samples.AspNetCore.Serilog` uses a different DSN
https://github.com/getsentry/sentry-dotnet/blob/0e11682db1da08cf4052525e528328197bc26354/samples/Sentry.Samples.AspNetCore.Serilog/appsettings.json#L6

Is this intentional, or should we use the same DSN as with all the other samples there too?

Reply:
Not intentional I don't think.

The only reason we have the DSN hard coded into the samples is to make it easier for us as the maintainers to fire up the samples and test things out. We could potentially remove all the DSNs and do that via an environment variable instead:
https://github.com/getsentry/sentry-dotnet/blob/1aad9ab9b432d9dc3118b77aa9d3002a2767a451/src/Sentry/Internal/SettingLocator.cs#L42-L43

That would prevent something like this from happening in future as well.

@bruno-garcia thoughts?

Reply:
yeah that's a good idea, if it'll work that way in all app types I dunno